index.html

```diff
@@ -20,13 +20,13 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
         <h1 class="detail">I’m a software developer</h1>
         <p class="pOne">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    </section>
-    <section id="socials">
-        <a href="#"><img src="./src/twitter.png" alt="twitter"></a>
-        <a href="#"><img src="./src/linkedin.png" alt="linkedin"></a>
-        <a class="mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
-        <a href="#"><img src="./src/github.png" alt="github"></a>
-        <a href="#"><img src="./src/angellist.png" alt="angellist"></a>
+        <div id="socials">
+        <a class="logos" href="#"><img src="./src/twitter.png" alt="twitter"></a>
+        <a class="logos" href="#"><img src="./src/linkedin.png" alt="linkedin"></a>
+        <a class="logos mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
+        <a class="logos" href="#"><img src="./src/github.png" alt="github"></a>
+        <a class="logos" href="#"><img src="./src/angellist.png" alt="angellist"></a>
+        </div>
     </section>
     <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```

Comment:
### Optional
- It would be better not to have a lot of `<br>` tags, it is better to create an empty div and give the spacing you desire

Reply:
This space is only temporary to check the sticky navbar however I'll remove them right away or change them to an empty div. Thanks!